environment.yml

```diff
@@ -20,6 +21,5 @@ dependencies:
   - pip:
       - typeshed-client==2.3.0
       - segmentation-models-pytorch==0.3.3
```

Comment:
Will install `segmentation-models-pytorch=0.3.3` from conda-forge once https://github.com/conda-forge/segmentation-models-pytorch-feedstock/pull/18 is merged.